client/components/account-status/account-fees/account-fees.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
I think we finalized `client/types` as the location to store all .d.ts files.

Reply:
Fixed in 1e25ce1e4cd7dd41fa1a9ef22e996614a682b9e7